src/main/kotlin/org/opensearch/replication/task/shard/TranslogSequencer.kt

```diff
@@ -73,26 +88,52 @@ class TranslogSequencer(scope: CoroutineScope, private val replicationMetadata:
                     var relativeStartNanos  = System.nanoTime()
                     val retryOnExceptions = ArrayList<Class<*>>()
                     retryOnExceptions.add(MappingNotAvailableException::class.java)
+                    var tooManyRequestsAcknowledgement = true
+                    try {
+                        while (tooManyRequestsAcknowledgement) {
```

Comment:
For the current changes, let's retry after the backoff and handle any errors within the translog sequencer.

Reply:
Ack